properties/basic.py

```diff
@@ -110,7 +131,7 @@ def assert_valid(self, instance):
                 )
             )
         if value is not None:
-            self.validate(instance, value)
+            instance._backend[self.name] = self.validate(instance, value)
```

Comment:
I would like to assume that people go through our interface rather than changing the `_backend` directly.

I would like some clarification on the `undefined` story. Can things be stored as `None`? Do these serialize --> `null` in json? If not always, then when?